src/dwarf.cpp

```diff
@@ -1380,6 +1394,24 @@ pool_string dwarf::implementation::resolve_type(attribute type) {
         result = empool("const " + recurse(attributes).allocate_string());
     } else if (die._tag == dw::tag::pointer_type) {
         result = empool(recurse(attributes).allocate_string() + "*");
+    } else if (die._tag == dw::tag::typedef_) {
```

Comment:
This is the major fix- an improvement to the way anonymous typedefs are handled. I think this resolves #84, but I'd like to add a test before declaring victory here.

Reply:
Update: This is not the fix for 84.